examples/maps/README.md

```diff
@@ -0,0 +1,40 @@
+# MapReader and maps
```

Comment:
I'll move some of the content I had shifted over to the Binder back here to this maps subpage. Will work on this page after this version is pushed.

Reply:
👍 

To be consistent, let's keep the content of the README as:

```
- [Tutorials](#tutorials)
  - [classification_one_inch_maps_001](https://github.com/Living-with-machines/MapReader/tree/main/examples/maps/classification_one_inch_maps_001)
- OTHER SECTIONS/COMMENTS/TEXT, e.g.:
- [Why use MapReader?](#why-use-mapreader)
- [Guidance for specific user groups](#guidance-for-specific-user-groups)
```

Reply:
Sure, I think that will work.